openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -181,7 +183,10 @@ def save(self,  # pylint: disable=arguments-differ
         """
         for user in self.users_cache:
             if self.is_account_recovery:
-                send_password_reset_email_for_user(user, request)
+                if is_testing_environment():
+                    send_password_reset_email_for_user(user, request)
+                else:
+                    compose_and_send_adg_password_reset_email(user, request)
```

Comment:
Same there is one more occurance `edx-platform/openedx/core/djangoapps/user_authn/views/login.py:128`

My suggestion would be to inject out code inside the function, if possible.

Reply:
I checked it for both flows i-e from registration:forgot_password and from settings:change_password, it's working for both.